phpseclib/Net/SFTP.php

```diff
@@ -430,7 +430,7 @@ public function __construct($host, $port = 22, $timeout = 10)
      * Login
      *
      * @param string $username
-     * @param string[] ...$args
```

Comment:
So technically, the second parameter can also be ]a \phpseclib3\Crypt\Common\PrivateKey object](https://phpseclib.com/docs/auth#public-key). It could also be an [array of arrays](https://phpseclib.com/docs/auth#keyboard-interactive)

Reply:
Made some adjustments. Hope I understood your intention correctly